include/flatbuffers/base.h

```diff
@@ -88,6 +88,12 @@
   #endif
 #endif
 
+#if defined(_MSC_VER) && (_MSC_VER < 1900)
+  #ifndef constexpr
+    #define constexpr const
```

Comment:
We already have a `FLATBUFFERS_CONSTEXPR` elsewhere

Reply:
Yes, I missed this.  
I will check with `FLATBUFFERS_CONSTEXPR` definition under Clang.

Should this code be removed as obsolete?
> Backwards compatability for g++ 4.4, and 4.5 which don't have the nullptr and constexpr keywords.

https://github.com/google/flatbuffers/blob/751a3d0f7804b97a423117c56926180c92125134/include/flatbuffers/base.h#L72-L90

Reply:
Why would we remove it? I'm not sure if we're still needing to support GCC 4.4, but maybe keep it in until we have stronger reasons not to support it.

Reply:
@aardappel 
I have checked (https://godbolt.org/z/eC-vEE).
The `-std=c++11` is available since gcc 4.7.1.
It is impossible to use C++11 without this specifier.
GCC 4.7.1 and higher have `nulptr_t`. Moreover type `nulptr_t` not used in code.
The `constexpr` not used directly also. Instead of it has `FLATBUFFERS_CONSTEXPR`.

Therefore this code is redundant.